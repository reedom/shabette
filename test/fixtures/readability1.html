<article>
  <h2>The <span>article</span> title</h2>
  <p>The first sentence. And second.
    <span><strong>T<code>hird comes here.</code></strong></span>
    <div>Fourth.</div>
  </p>
  Fifth.
</article>
