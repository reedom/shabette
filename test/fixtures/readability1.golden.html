<html><head></head><body><article>
  <h2><shabette-sentence class="shabette-sentence-1">The </shabette-sentence><span><shabette-sentence class="shabette-sentence-1">article</shabette-sentence></span><shabette-sentence class="shabette-sentence-1"> title</shabette-sentence></h2>
  <p><shabette-sentence class="shabette-sentence-3">The first sentence. </shabette-sentence><shabette-sentence class="shabette-sentence-4">And second.
</shabette-sentence><shabette-sentence class="shabette-sentence-5"> </shabette-sentence><shabette-sentence class="shabette-sentence-6"> </shabette-sentence><shabette-sentence class="shabette-sentence-7"> </shabette-sentence><shabette-sentence class="shabette-sentence-8"> </shabette-sentence><span><strong><shabette-sentence class="shabette-sentence-8">T</shabette-sentence><code><shabette-sentence class="shabette-sentence-8">hird comes here.</shabette-sentence></code></strong></span>
    </p><div><shabette-sentence class="shabette-sentence-10">Fourth.</shabette-sentence></div>
  <p></p><shabette-sentence class="shabette-sentence-12">
  Fifth.
</shabette-sentence></article>
</body></html>
